src/LanguageServer.php

```diff
@@ -206,6 +206,13 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
                 // Find composer.json
                 if ($this->composerJson === null) {
                     $composerJsonFiles = yield $this->filesFinder->find(Path::makeAbsolute('**/composer.json', $rootPath));
+
+                    // If we sort our findings by string length (shortest to longest),
+                    // the first entry will be the project's root composer.json.
```

Comment:
I don't see a `Uri` class included.  Do you have a parsing library to recommend adding to the project?